lib/ansible/module_utils/vmware_rest_client.py

```diff
@@ -49,6 +50,22 @@ def __init__(self, module):
         self.check_required_library()
         self.api_client = self.connect_to_vsphere_client()
 
+    # Helper function
+    def get_error_message(self, error):
```

Comment:
Hi Abhijeet,
Checking Error stack provided in issue https://github.com/ansible/ansible/issues/58326 
`[LocalizableMessage(default_message='MYDOMAIN.COM\\\\service_account does not have attach or detach privilege on urn:vmomi:InventoryServiceTag:bcf7a240-1dda-43ed-a98e-dfb9806d1849:GLOBAL`
Error message looks obvious and it is very clear to the user, what is the issue.  it is clearly credentials privilege issue. I believe change is not really required. Please correct me if I am wrong.

Reply:
Yes, it does provide the error message in stack, but it lacks better UI/UX. I feel, this fix adds a better user experience.

Reply:
With this change we can see -

```
TASK [Assign tag from inventory] ***************************************************************************
fatal: [VM_8044 -> localhost]: FAILED! => {"changed": false, "msg": "VSPHERE.LOCAL\\ansiboy does not have attach or detach privilege on urn:vmomi:InventoryServiceTag:43737ec0-b832-4abf-abb1-fd2448ce3b26:GLOBAL"}
```

Reply:
got it.  LGTM

Thanks